nixos/modules/virtualisation/virtualbox-guest.nix

```diff
@@ -27,6 +49,33 @@ in
       type = types.bool;
       description = lib.mdDoc "Whether to enable x11 graphics";
     };
+
+    conditionVirtualization = mkOption {
+      default = "oracle";
+      type = types.str;
+      description = lib.mdDoc ''
+        The virtualized environment in which the guest additions should be started.
+        E.g., "oracle" or "kvm"
+      '';
+    };
+
+    clipboard = mkOption {
+      default = true;
+      type = types.bool;
+      description = lib.mdDoc "Whether to enable clipboard support";
+    };
+
+    seamless = mkOption {
+      default = true;
+      type = types.bool;
+      description = lib.mdDoc "Whether to enable seamless support";
+    };
+
+    vmsvga = mkOption {
```

Comment:
nit: For these options, is there any serious usecase where you would want to disable them from the guest perspective? I would also be fine with these being enabled all the time to simplify the module. These are typically configured from the host anyhow.